src/containers/CardList.js

```diff
@@ -56,9 +56,8 @@ class CardList extends Component {
           data={data}
           handleDropdownChange={(e, value) => this.handleDropdownChange(value)}
         />
-        <Carousel>
-          {funcCards}
-        </Carousel>
+        <Carousel children={funcCards} />
```

Comment:
This doesn't change anything :disappointed: It doesn't matter how you set the children property, it will end up as props.children in Carousel component. The final result is the same.

Reply:
It doesn't work even if I replace everything in App with example code from https://react-materialize.github.io/#/carousel.